### PR TITLE
HUM-896 : after stabilizing an item- notify its peers

### PR DESCRIPTION
**Pull request by dpgoetz**

### CODE REVIEW

File: objectserver/ecengine.go [hunk 1]
```diff
@@ -465,43 +534,95 @@ func (f *ecEngine) RegisterHandlers(addRoute func(method, path string, handler h
...
+	go idb.ListObjectsToStabilize(idbItemsChan, cancel)
+	allDone := false
+	for !allDone {
+		objs := []*ecObject{}
+		for i := 0; i < numStabilizeObjects; i++ {
+			item := <-idbItemsChan
+			if item == nil {
+				allDone = true
+				break
+			} else {
```
Comment (corystone) [inline objectserver/ecengine.go]: I would de-indent this.

File: objectserver/ecengine.go [hunk 2]
```diff
@@ -465,43 +534,95 @@ func (f *ecEngine) RegisterHandlers(addRoute func(method, path string, handler h
...
-			nurseryReplicas: f.nurseryReplicas,
-			txnId:           fmt.Sprintf("%s-%s", common.UUID(), device),
+	idbItemsChan := make(chan *IndexDBItem, numStabilizeObjects)
+	cancel = make(chan struct{})
+	defer close(cancel)
+	go idb.ListObjectsToStabilize(idbItemsChan, cancel)
+	allDone := false
+	for !allDone {
+		objs := []*ecObject{}
+		for i := 0; i < numStabilizeObjects; i++ {
```
Comment (corystone) [inline objectserver/ecengine.go]: It's kind of strange using this here AND as a db query limit. Might not be a huge deal, but if db1 has 1000 things, you'll only do those and stop? Maybe that's exactly what's intended...

File: objectserver/ecobj.go [hunk 3]
```diff
@@ -432,6 +436,41 @@ func (o *ecObject) Replicate(prirep PriorityRepJob) error {
...
+				return
+			}
+			defer resp.Body.Close()
+			if resp.StatusCode/100 == 2 {
+				atomic.AddInt64(&successes, 1)
+			}
+		}(i, node)
+	}
+	wg.Wait()
+	o.logger.Debug("stabilze notification", zap.Bool("allNodes", successes == int64(len(nodes))))
```
Comment (corystone) [inline objectserver/ecobj.go]: Does this count itself?

**Comment by nadeemsyed** [comment]
LGTM!

**Pull request merged**